src/KubernetesClient/Config.php

```diff
@@ -530,17 +538,25 @@ protected function getAuthProviderToken()
             throw new \Error("error retrieving token: auth provider failed to return valid data");
```

Comment:
Should this be `Exception`?

Reply:
I just continued using the class `Error` that was in use for raising problems. It is a `throwable` alike `Exception` and I just learned that PHP has such a class from your code. So I have to pass the question back to you. To me it seems as if class `Error` is quite suitable, since all this error cases prevent successful operation (since no auth can be aquired).

Reply:
That’s fair. I don’t see anywhere in my code that I’m throwing an error but the finality of the situation does make sense as you say. Let’s leave it.